lib/eco_service_web/live/eco_service_live/community_list.html.heex

```diff
@@ -1,14 +1,35 @@
+<h1 class="font-bold text-lg text-center">List of communities</h1>
 <.table id="community" rows={@communities}>
-<:col :let={community} label="Community Name"> <%= community.name%> </:col>
-<:col :let={community} label="Location Area Zone"> <%= community.location_area_zone%> </:col>
-<:col :let={community} label="Edit"> 
-    <.link phx-click="delete" phx-value-id={community.id}> <span class="text-blue-500">Delete</span> </.link>    
-</:col>
+  <:col :let={community} label="Community Name"> <%= community.name%> </:col>
+  <:col :let={community} label="Location Area Zone"> <%= community.location_area_zone%> </:col>
+  <:col :let={community} label="Delete"> 
+      <.link phx-click="delete" phx-value-id={community.id} > <span class="text-blue-500">Delete</span> </.link>    
+  </:col>
+  <:col :let={community} label="Edit"> 
+      <.link href={~p"/community/#{community.id}/edit"} > <span class="text-blue-500">Edit</span> </.link>    
+  </:col>
 </.table>
  
 <%= if @offset != 0 do %>  
 <.link phx-click="prev" ><p class="text-blue-500">Prev</p></.link>
 <% end%>
 <%= if @all_records - @offset > @limit do %>
 <.link phx-click="next" ><p class="text-blue-500">Next</p></.link>
-<% end %>
\ No newline at end of file
+<% end %>
+
+
+<.modal
+  :if={@live_action in [:edit]} 
```

Comment:
The `in` operator is particularly useful if there are multiple items in the list. The list that you have, contains only one item. How do you think you can write this better?

Reply:
= ?